### ISSUE TITLE
[master] "Invalid literal for int() with base 10: 'dev1'"

### ISSUE THREAD

**Issue by dan-blanchard** [pytest-dev/pytest#1312]
To work around the fact that there isn't a release with the #1142 fix in it, I've been installing master.  However, when I installed it again today in a different virtualenv, I ran into this new issue:

```pytb
$ py.test -v
Traceback (most recent call last):
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/bin/py.test", line 9, in <module>
    load_entry_point('pytest', 'console_scripts', 'py.test')()
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/config.py", line 38, in main
    config = _prepareconfig(args, plugins)
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/config.py", line 117, in _prepareconfig
    pluginmanager=pluginmanager, args=args)
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/vendored_packages/pluggy.py", line 724, in __call__
    return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/vendored_packages/pluggy.py", line 338, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/vendored_packages/pluggy.py", line 333, in <lambda>
    _MultiCall(methods, kwargs, hook.spec_opts).execute()
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/vendored_packages/pluggy.py", line 595, in execute
    return _wrapped_call(hook_impl.function(*args), self.execute)
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/vendored_packages/pluggy.py", line 249, in _wrapped_call
    wrap_controller.send(call_outcome)
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/helpconfig.py", line 28, in pytest_cmdline_parse
    config = outcome.get_result()
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/vendored_packages/pluggy.py", line 279, in get_result
    _reraise(*ex)  # noqa
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/vendored_packages/pluggy.py", line 264, in __init__
    self.result = func()
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/vendored_packages/pluggy.py", line 596, in execute
    res = hook_impl.function(*args)
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/config.py", line 852, in pytest_cmdline_parse
    self.parse(args)
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/config.py", line 957, in parse
    self._preparse(args)
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/config.py", line 918, in _preparse
    self.pluginmanager.load_setuptools_entrypoints("pytest11")
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/src/pytest/_pytest/vendored_packages/pluggy.py", line 501, in load_setuptools_entrypoints
    plugin = ep.load()
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2380, in load
    return self.resolve()
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2386, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/dan/.pyenv/versions/miniconda3-3.16.0/envs/casterisk/lib/python2.7/site-packages/hypothesis/extra/pytestplugin.py", line 21, in <module>
    PYTEST_VERSION = tuple(map(int, pytest.__version__.split('.')))
ValueError: invalid literal for int() with base 10: 'dev1
```

**Comment by The-Compiler** [comment]
As you can see from a traceback, that's a bug in hypothesis checking the pytest version - I think it was already fixed though. Are you on some old release?

Either way I'm closing this, as it's definitely not a pytest issue.

/cc @DRMacIver

**Comment by DRMacIver** [comment]
Yes, this has been fixed for a while, though I can't seem to find the bug in the change log. At any rate it should work fine in the latest version.

**Comment by DRMacIver** [comment]
Looks like it was fixed in Hypothesis 1.11.2, but you should definitely just upgrade to the latest anyway :-)

**Comment by dan-blanchard** [comment]
Sorry! I just created this virtualenv yesterday, so I'm not sure how I ended up with old versions.  I'll look into that.
